src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -123,4 +123,29 @@ public void doGet(
     }
     return events;
   }
+
+  /**
+   * Creates a new event when POST method called. Use the time boundaries given in the request for
+   * creating the event in the primary calendar. The summary for this event is Read emails.
+   *
+   * @param request Http request from the client. Body contains start and end parameters
+   * @param response 403 if user is not authenticated, or string saying the event has been created
```

Comment:
Documentation: you are not actually returning 403 in the response here, and this method won't be called if a 403 is returned by the base class.

Reply:
`Resolved`